controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -136,11 +149,122 @@ func (r *SriovNetworkNodePolicyReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 		return reconcile.Result{}, err
 	}
 
+	if err = r.syncOffloadMachineConfig(policyList); err != nil {
```

Comment:
this will break deployment in vanilla k8s

Reply:
I'll introduce an environemnt variable to differentiate openshift and vanilla k8s.